evaluator/bundle.go

```diff
@@ -69,3 +71,18 @@ func StartServer(ctx context.Context, cfg config.Config) (*http.Server, error) {
 
 	return srv, nil
 }
+
+// loadBundle loads the relevant bundle based on the config.
+func loadBundle(cfg config.Config, log *logp.Logger) csppolicies.Bundle {
```

Comment:
Wondering about this method.

Do we wish to throw an error if we don't receive any of the two?

WDT?

Reply:
good point. 
I had the impression that we would want to have vanilla as default in any case the config has some misconfiguration as we had before.
Additionally, keeping it like that will be backward compatible with the previous config format, which is still what we have today, and will not break the integration test. We can update this function to be more strict once we have working integration between the config and cloudbeat.